app/src/main/java/org/thunderdog/challegram/unsorted/Settings.java

```diff
@@ -6074,6 +6079,39 @@ public int getEmojiPackInstallState (CloudSetting setting, boolean fast) {
     return hasFile ? (installedVersion == setting.date ? CloudSetting.STATE_INSTALLED : CloudSetting.STATE_UPDATE_NEEDED) : CloudSetting.STATE_NOT_INSTALLED;
   }
 
+  private String[] quickReactions;
+  public void setQuickReactions (String reactions[]) {
+    pmc.edit().putStringArray(KEY_QUICK_REACTIONS, reactions).apply();
```

Comment:
```suggestion
    pmc.putStringArray(KEY_QUICK_REACTIONS, reactions);
```

LevelDB object doesn't require prior call to `edit()`, it is needed only for batch operations.